src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
+
+##### Message Bundle Keys and Selectors #####
+
+######Additions######
+* `textSizeDescr`
+* `textFontDescr`
+* `lineSpaceDescr`
+* `contrastDescr`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>speakChoiceLabel</code></td>
+    <td><code>speakDescr</code></td>
+</tr>
+<tr>
+    <td><code>tocChoiceLabel</code></td>
+    <td><code>tocDescr</code></td>
+</tr>
+<tr>
+    <td><code>simplifyChoiceLabel</code></td>
+    <td><code>simplifyDescr</code></td>
+</tr>
+</table>
+
+##### CSS Selectors #####
```

Comment:
@jobara, I was initially going to agree, and say we should just call the section "Selectors" since it's refering to an option called "selectors", but then I had a look at the file as a whole to be sure I knew what I was talking about, and I realized I can't tell what this section is saying. The "CSS Selectors" heading is a sibling to the "Message Bundle Keys and Selectors" – what is the difference in selectors? The "CSS Selectors" heading is followed by "additions" and a list of selectors, but what are they? Don't our DOM selectors have names? What are the names for these selectors? Am I even right that this is referring to the 'selectors' option?